homeassistant/components/circuit/notify.py

```diff
@@ -0,0 +1,43 @@
+"""Unify Circuit platform for notify component."""
+import logging
+
+from circuit_webhook import Circuit
+import voluptuous as vol
+
+from homeassistant.components.notify import PLATFORM_SCHEMA, BaseNotificationService
+from homeassistant.const import CONF_URL
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_URL): cv.url})
```

Comment:
Please set up the integration from a config schema in the integration init module. Use a `CONFIG_SCHEMA`, and load the notify platform via the discovery helper `load_platform`.

See other integrations that uses `load_platform` for examples.

Also see ADR 7:
https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md

Reply:
Thank you for your suggestion but I think it's not necessary as the component only send outgoing messages and do not accept incoming messages. Circuit from Unify is actually almost the same as MS Teams. So my code is heavily based on the msteams component.

For more than one webhook I tested successfully:
notify:
  - platform: circuit
    name: circuit_webhook_1
    url: https://xxx/ID_1
  - platform: circuit
    name: circuit_webhook_2
    url: https://xxx/ID_2

Reply:
We won't accept this PR in its current state. Please follow my request above.

Reply:
Thanks for your quick reply, I looked into the documentation and found this example: https://github.com/home-assistant/example-custom-config/tree/master/custom_components/example_load_platform/ 

Is that what you are requesting? The configuration.yaml will then be changed to:
circuit:
  name: circuit_webhook_1
  url: https://xxx/ID

Reply:
Yes. Make sure to add a `CONFIG_SCHEMA` in the init module of the integration. That's not in the example.

Reply:
New commit added with code changes as requested. I was sruggling a bit with the logic to handle this in the configuration.yaml file. I ended up with new config layout:

```
# Example configuration.yaml entry
circuit:
  - webhook:
      - name: webhook_1
        url: "https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>"
      - name: webhook_2
        url: "https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>"
      - name: webhook_multiple
        url: 
          - "https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-1>"
          - "https://eu.yourcircuit.com/rest/v2/webhooks/incoming/<ID-2>"   
```